_includes/pbc/all/install-features/202505.0/install-the-order-amendment-feature.md

```diff
@@ -0,0 +1,1296 @@
+
+Make sure that `mysprykershop.com/order-amendment` with a guest user redirects to login page.
+
+{% endinfo_block %}
+
+**src/Pyz/Client/MultiCart/MultiCartConfig.php**
+
+```php
+<?php
+
+namespace Pyz\Client\MultiCart;
+
+use Generated\Shared\Transfer\QuoteTransfer;
+use Spryker\Client\MultiCart\MultiCartConfig as SprykerMultiCartConfig;
+
+class MultiCartConfig extends SprykerMultiCartConfig
+{
+    /**
+     * @return array<string|array<string>>
+     */
+    public function getQuoteFieldsAllowedForCustomerQuoteCollectionInSession(): array
+    {
+        return array_merge(parent::getQuoteFieldsAllowedForCustomerQuoteCollectionInSession(), [
+            QuoteTransfer::AMENDMENT_ORDER_REFERENCE
+        ]);
+    }
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure that only configured fields are saved in a customer's session.
+
+{% endinfo_block %}
+
+**src/Pyz/Zed/Quote/QuoteConfig.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\Quote;
+
+use Generated\Shared\Transfer\QuoteTransfer;
+use Spryker\Zed\Quote\QuoteConfig as SprykerQuoteConfig;
+
+class QuoteConfig extends SprykerQuoteConfig
+{
+    /**
+     * @return array
+     */
+    public function getQuoteFieldsAllowedForSaving()
+    {
+        return array_merge(parent::getQuoteFieldsAllowedForSaving(), [
+            QuoteTransfer::AMENDMENT_ORDER_REFERENCE,
+            QuoteTransfer::QUOTE_PROCESS_FLOW,
+        ]);
+    }
+}
+```
+
+{% info_block warningBox “Verification” %}
+
+Make sure that when you edit order, JSON data in the database column `spy_quote.quote_data` of the corresponding quote
+contains `amendmentOrderReference` and `quoteProcessFlow`.
+
+{% endinfo_block %}
+
+**src/Pyz/Zed/SalesOrderAmendment/SalesOrderAmendmentConfig.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\SalesOrderAmendment;
+
+use Generated\Shared\Transfer\QuoteTransfer;
+use Spryker\Zed\SalesOrderAmendment\SalesOrderAmendmentConfig as SprykerSalesOrderAmendmentConfig;
+
+class SalesOrderAmendmentConfig extends SprykerSalesOrderAmendmentConfig
+{
+    /**
+     * @return array<string>
+     */
+    public function getQuoteFieldsAllowedForSaving(): array
+    {
+        return array_merge(parent::getQuoteFieldsAllowedForSaving(), [
+            QuoteTransfer::BUNDLE_ITEMS,
+            QuoteTransfer::CART_NOTE,
+            QuoteTransfer::EXPENSES,
+            QuoteTransfer::VOUCHER_DISCOUNTS,
+            QuoteTransfer::GIFT_CARDS,
+            QuoteTransfer::CART_RULE_DISCOUNTS,
+            QuoteTransfer::PROMOTION_ITEMS,
+            QuoteTransfer::IS_LOCKED,
+            QuoteTransfer::QUOTE_REQUEST_VERSION_REFERENCE,
+            QuoteTransfer::QUOTE_REQUEST_REFERENCE,
+            QuoteTransfer::MERCHANT_REFERENCE,
+            QuoteTransfer::IS_ORDER_PLACED_SUCCESSFULLY,
+        ]);
+    }
+}
+```
+
+{% info_block warningBox “Verification” %}
+
+Table `spy_sales_order_amendment_quote` is not used yet. This table could be used as a temporary storage for the quote
```

Comment:
Are the data saved already to this table? If yes I think it should be mentioned in verification with a note that this data is not used yet.